lib/BlockedBy.js

```diff
@@ -260,12 +260,16 @@ class BlockedBy extends NodeResque.Plugin {
         const buildsWaiting = await this.queueObject.connection.redis.lrange(waitingKey, 0, -1);
         const keyExist = buildsWaiting.some(key => parseInt(key, 10) === buildId);
 
+        let statusMessage = 'Blocked by these running build(s): ';
+
+        blockingBuildIds.forEach((blockingBuildId) => {
+            statusMessage += `<a href="/builds/${blockingBuildId}">${blockingBuildId}</a> `;
```

Comment:
[IMO] It has a littel needless trailing space. And contatenating buildIds with space is a little hard to see. How about use `map` function and joining by ',' instead of `forEach` function like below.
```
> blockingBuildIds = [123,456,789]
[ 123, 456, 789 ]
> let statusMessage = 'Blocked by these running build(s): ';
undefined
> statusMessage += blockingBuildIds.map(blockingBuildId =>`<a href="/builds/${blockingBuildId}">${blockingBuildId}</a>`).join(', ');
'Blocked by these running build(s): <a href="/builds/123">123</a>, <a href="/builds/456">456</a>, <a href="/builds/789">789</a>'
```

Reply:
Thank you for the review. That's a good idea.
I fix from now on.